src/components/CohortBuilder/Categories/index.js

```diff
@@ -128,17 +137,36 @@ class Categories extends React.Component {
     this.props.onSqonUpdate(...args);
   }
 
+  // searching should not open quick filters
   handleSearchField(fieldName) {
-    this.setState({ currentSearchField: fieldName });
+    let currentCategory = null;
+    if (CATEGORY_FIELDS.clinical.indexOf(fieldName) > -1) {
+      currentCategory = CATEGORY_NAMES.clinical;
+    } else if (CATEGORY_FIELDS.study.indexOf(fieldName) > -1) {
+      currentCategory = CATEGORY_NAMES.study;
+    } else if (CATEGORY_FIELDS.biospecimen.indexOf(fieldName) > -1) {
+      currentCategory = CATEGORY_NAMES.biospecimen;
+    } else if (CATEGORY_FIELDS.demographic.indexOf(fieldName) > -1) {
+      currentCategory = CATEGORY_NAMES.demographic;
+    } else if (CATEGORY_FIELDS.availableData.indexOf(fieldName) > -1) {
+      currentCategory = CATEGORY_NAMES.availableData;
+    }
+    this.setState({ currentSearchField: fieldName, currentCategory });
   }
 
   handleCategoryClose() {
-    this.setState({ currentSearchField: '' });
+    this.setState({ currentSearchField: '', currentCategory: null });
```

Comment:
I think you should use `setActiveCategory` from everywhere you set the state of both `currentSearchField` and `currentCategory`.